### PR TITLE
Rules modify

### PR DESCRIPTION
**Pull request by vvikramb**
Rules modify

### CODE REVIEW

File: juniper_official/chassis/check-psm-power-usage-state.rule [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+ *   3) "psm" is the power module,it can be psm0 or psm1.
  *      
  */
 healthbot {
     topic hardware.chassis {
         rule check-psm-power-usage-state {
             /*
              * Monitors PSM power usage,state and notifies via dashboard, red color 
-             * when threshold is exceeded else green.
+             * when high threshold is exceeded,yellow when low is exceeded else green.
```
Comment (gvarao) [inline juniper_official/chassis/check-psm-power-usage-state.rule:18]: Red is missing in comment

File: juniper_official/chassis/check-fan-state-rpm.rule [hunk 2]
```diff
@@ -135,17 +135,17 @@ healthbot {
...
-                        greater-than "$rpm-percent" "$high-threshold";
-                    }
+                        less-than-or-equal-to "$rpm-percent" "$low-threshold";
+                    }				
                     then {
                         status {
-                            color red;
-                            message "$fan-name RPM percent $rpm-percent($measurement RPM) is greater than $rpm-high-threshold";
+                            color green;
+                            message "$fan-name RPM percent is $rpm-percent";
```
Comment (gvarao) [inline juniper_official/chassis/check-fan-state-rpm.rule:148]: As it is normal. Please add normal in message e.g. $fan-name RPM percent ($rpm-percent) is normal

File: juniper_official/chassis/check-psm-power-usage-state.rule [hunk 3]
```diff
@@ -91,41 +98,58 @@ healthbot {
...
                     when {
-                        does-not-match-with "$psm-state" offline {
-                            ignore-case;
+                        greater-than-or-equal-to "$psm-power-usage" "$psm-power-usage-low-threshold";
+                        less-than "$psm-power-usage" "$psm-power-usage-high-threshold";
+                    }
+                    then {
+                        status {
+                            color yellow;
+                            message "$psm power usage($psm-power-usage) exceeds low threshold (50)";
```
Comment (gvarao) [inline juniper_official/chassis/check-psm-power-usage-state.rule:130]: Try to follow similarity/format of thresholds in all rule's of all trigger messages. I see dynamic values in some rules and static in some rule's messages. 

e.g. 
Static:
message "$psm power usage($psm-power-usage) exceeds low threshold (50)";
Dynamic:
message "$fan-name RPM percent $rpm-percent($measurement RPM) is greater than $rpm-high-threshold"

**Review by gvarao — CHANGES_REQUESTED** [review]

File: juniper_official/chassis/check-psm-power-usage-state.rule (hunk 1 above)
Comment (vvikramb) [reply juniper_official/chassis/check-psm-power-usage-state.rule:18]: Red color is included :
             * Monitors PSM power usage,state and notifies via dashboard, red color 
             * when high threshold is exceeded,yellow when low is exceeded else green.

File: juniper_official/chassis/check-fan-state-rpm.rule (hunk 2 above)
Comment (vvikramb) [reply juniper_official/chassis/check-fan-state-rpm.rule:148]: Done

File: juniper_official/chassis/check-psm-power-usage-state.rule (hunk 3 above)
Comment (vvikramb) [reply juniper_official/chassis/check-psm-power-usage-state.rule:130]: Changed to dynamic value.

File: juniper_official/chassis/check-psm-power-usage-state.rule [hunk 4]
```diff
@@ -91,41 +98,58 @@ healthbot {
...
-                 * Sets color to red when psm-power-usage is greater than psm-power-usage-threshold
+                 * Sets color to green when psm-power-usage is lesser than psm-power-usage-low-threshold
                  * 
                  */				
-                term is-power-usage-abnormal {
+                term power-usage-normal {
                     when {
-                        greater-than-or-equal-to "$psm-power-usage" "$psm-power-usage-threshold";
+                        less-than "$psm-power-usage" "$psm-power-usage-low-threshold";
+                        does-not-match-with "$psm-state" offline {
```
Comment (gaddevijay) [inline juniper_official/chassis/check-psm-power-usage-state.rule:107]: Wouldn't it be better to compare with what we we are expecting?

File: juniper_official/chassis/check-routing-engine-temperature.rule [hunk 5]
```diff
@@ -37,10 +37,16 @@ healthbot {
...
                  description "open-config sensor to collect telemetry data from network device";
                  open-config {
-                     sensor-name /components/;
+                     sensor-name /components/component;
                      frequency 60s;
                  }
              }
+             sensor components-oc-others {
+                 open-config {
+                     sensor-name /components/;
```
Comment (gaddevijay) [inline juniper_official/chassis/check-routing-engine-temperature.rule:46]: This also should be /components/component

File: juniper_official/interface/check-optical-temp-thresholds.rule [hunk 6]
```diff
@@ -70,19 +70,19 @@ healthbot {
...
                     when {
-                        greater-than-or-equal-to "$optical-temp" "$high-alarm-threshold";
-                    }
+                        less-than "$optical-temp" "$high-warning-threshold";
+                    }				
                     then {
                         status {
-                            color red;
-                            message "$interface-name,temperature $optical-temp exceeds default alarm threshold ($high-alarm-threshold)";
+                            color green;
```
Comment (gaddevijay) [inline juniper_official/interface/check-optical-temp-thresholds.rule:81]: All yellow thresholds needs to rechecked where only green and red are changed

**Review by gaddevijay — APPROVED** [review]

File: juniper_official/chassis/check-routing-engine-temperature.rule (hunk 5 above)
Comment (vvikramb) [reply juniper_official/chassis/check-routing-engine-temperature.rule]: The sensor "components-oc-others"" will be removed as RE temperature path will be "temperature" now across all models.

File: juniper_official/interface/check-optical-temp-thresholds.rule (hunk 6 above)
Comment (vvikramb) [reply juniper_official/interface/check-optical-temp-thresholds.rule:81]: The latest rule in jcloud has the updated changes, it was merged as part of another MR.

File: juniper_official/chassis/check-psm-power-usage-state.rule (hunk 4 above)
Comment (vvikramb) [reply juniper_official/chassis/check-psm-power-usage-state.rule:107]: Changed to Online the new MR.

**Pull request closed without merging**